module/version_reconfigure_test.go

```diff
@@ -29,21 +30,25 @@ func TestValidationFailureDuringReconfiguration(t *testing.T) {
 		},
 		Components: []resource.Config{
 			{
-				Name:       "generic1",
-				Model:      resource.NewModel("acme", "demo", "multiversionmodule"),
-				API:        generic.API,
-				Attributes: utils.AttributeMap{},
+				Name:                "generic1",
+				Model:               resource.NewModel("acme", "demo", "multiversionmodule"),
+				API:                 generic.API,
+				Attributes:          utils.AttributeMap{},
+				ConvertedAttributes: &fakemotor.Config{},
 			},
 			{
-				Name:       "motor1",
-				Model:      resource.DefaultModelFamily.WithModel("fake"),
-				API:        motor.API,
-				Attributes: utils.AttributeMap{},
+				Name:                "motor1",
+				Model:               resource.DefaultModelFamily.WithModel("fake"),
+				API:                 motor.API,
+				Attributes:          utils.AttributeMap{},
+				ConvertedAttributes: &fakemotor.Config{},
 			},
 			{
-				Name:  "encoder1",
-				Model: resource.DefaultModelFamily.WithModel("fake"),
-				API:   encoder.API,
+				Name:                "encoder1",
+				Model:               resource.DefaultModelFamily.WithModel("fake"),
+				API:                 encoder.API,
+				Attributes:          utils.AttributeMap{},
+				ConvertedAttributes: &fakeencoder.Config{},
```

Comment:
Added due to explicit recommendation. I figured I'd add in `ConvertedAttributes` for the other parts to help avoid the question of which parts need an empty value and which are fine without.